cmd/cloud.go

```diff
@@ -344,8 +343,14 @@ This will execute the test on the k6 cloud service. Use "k6 login cloud" to auth
 				return errext.WithExitCodeIfNone(errors.New("Test progress error"), exitcodes.CloudFailedToGetProgress)
 			}
 
-			valueColor := getColor(globalState.flags.noColor || !globalState.stdOut.isTTY, color.FgCyan)
-			fprintf(globalState.stdOut, "     test status: %s\n", valueColor.Sprint(testProgress.RunStatusText))
+			if !globalState.flags.quiet {
+				valueColor := getColor(globalState.flags.noColor || !globalState.stdOut.isTTY, color.FgCyan)
```

Comment:
```suggestion
				valueColor := getColor(!globalState.UseColor(), color.FgCyan)
```
I would wrap this logic in a method. I think the name could be better from my suggestion

Reply:
The problem is that sometimes it's for `stdOut` and sometimes for `stdErr`, and sometimes we consider `noColor` and `isTTY` separately (e.g. in `cmd/ui.go`). Not saying that the current situation can't be improved, but I'll probably do it in a separate PR at this point.